server/src/utils/constants.js

```diff
@@ -0,0 +1,11 @@
+const DEFAULTS = {
+  auth_needed_to_vote: false,
+  auth_needed_to_write: true,
+  auth_opt_allow_3rdparty: true,
+  auth_opt_fb: true,
+  auth_opt_tw: true,
+};
```

Comment:
I like the standalone defaults / constants file. This makes it easier to configure these values later, separate from code.

Reply:
@colinmegill can confirm that these are the desired defaults. The logic all looks good to me.